src/searchForm/datePicker/datePickerFullScreen/index.tsx

```diff
@@ -0,0 +1,11 @@
+import styled from 'styled-components'
+
+import DatePickerFullScreen from './DatePickerFullScreen'
```

Comment:
As you know `DatePickerFullScreen` is now `DatePickerSection` and many other things have changed. If you need some help for the rebase feel free to call me ;)